easybuild/easyconfigs/e/elfutils/elfutils-0.185-GCCcore-10.3.0.eb

```diff
@@ -0,0 +1,41 @@
+easyblock = 'ConfigureMake'
+
+name = 'elfutils'
+version = '0.185'
+
+homepage = 'https://elfutils.org/'
+
+description = """
+ The elfutils project provides libraries and tools for ELF files
+ and DWARF data.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '10.3.0'}
+
+source_urls = ['https://sourceware.org/elfutils/ftp/0.183/']
```

Comment:
This `source_urls` is for a previous version?

Reply:
I have no idea how i managed to download the sources